docker-compose.helm.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.7'
+
+services:
+  helm3.14.4:
+    image: okteto/helm:3.14.4
+    build:
+      context: .
```

Comment:
Minor thing to possibly review in the future. Shouldn't the context be `helm`? @pchico83  Is there a reason why we use the whole main folder as build context for the images (I've checked that we are doing it in that way for all the images)?